contracts/src/v0.8/vrf/testhelpers/VRFV2WrapperLoadTestConsumer.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
```

Comment:
Are we creating these load test wrapper contracts just to control and validate the flow of the VRF request processing better?

Reply:
Nvm, this is just a consumer contract implementation. I have just realized this while reading the rest of the code.